RequestAsBrowserOptions TS type is missing options useInsecureHttpParser & ignoreSslErrors

TS type declarations of `RequestAsBrowserOptions` are missing options mentioned in https://sdk.apify.com/docs/api/utils#utilsrequestasbrowseroptions and if you look in the code, the options are cleary consumed and handled, but they're missing in the type declarations.

> IMPORTANT: This function uses an insecure version of HTTP parser by default and also ignores SSL/TLS errors. This is very useful in scraping, because it allows bypassing certain anti-scraping walls, but it also exposes some vulnerability. For other than scraping scenarios, please set useInsecureHttpParser: false and ignoreSslErrors: false.

**System information:**
 - OS: `Ubuntu 18.04.4 LTS`
 - Node.js version `v10.22.0`
 - Apify SDK version `0.21.3`, [yarn.lock.txt](https://github.com/apify/apify-js/files/5010301/yarn.lock.txt)

--- 

code:

```ts
import Apify from "apify";
import * as cheerio from "cheerio";

export async function test() {
    const {body} = await Apify.utils.requestAsBrowser({
        url: 'https://www.google.cz',
        useInsecureHttpParser: false,
        ignoreSslErrors: false
    });

    const $ = cheerio.load(body);
}
```

tsc output:

```
src/reproducer.ts:7:9 - error TS2345: Argument of type '{ url: string; useInsecureHttpParser: boolean; ignoreSslErrors: boolean; }' is not assignable to parameter of type 'RequestAsBrowserOptions'.
  Object literal may only specify known properties, and 'useInsecureHttpParser' does not exist in type 'RequestAsBrowserOptions'.

7         useInsecureHttpParser: false,
          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/src/utils_request.js
+++ b/src/utils_request.js
@@ -36,10 +36,6 @@ const DEFAULT_HTTP_REQUEST_OPTIONS = {
  *  Two-letter ISO 3166 country code.
  * @property {boolean} [useMobileVersion]
  *  If `true`, the function uses User-Agent of a mobile browser.
- * @property {AbortFunction} [abortFunction]
- *  Function accepts `response` object as a single parameter and should return true or false.
- *  If function returns true request gets aborted. This function is passed to the
- *  [@apify/http-request](https://www.npmjs.com/package/@apify/http-request) NPM package.
  * @property {boolean} [ignoreSslErrors=true]
  *  If set to true, SSL/TLS certificate errors will be ignored.
  * @property {boolean} [useInsecureHttpParser=true]
@@ -50,6 +46,10 @@ const DEFAULT_HTTP_REQUEST_OPTIONS = {
  *  However, it will also open your application to some security vulnerabilities,
  *  although the risk should be negligible as these vulnerabilities mainly relate to server applications, not clients.
  *  Learn more in this [blog post](https://snyk.io/blog/node-js-release-fixes-a-critical-http-security-vulnerability/).
+ * @property {AbortFunction} [abortFunction]
+ *  Function accepts `response` object as a single parameter and should return true or false.
+ *  If function returns true request gets aborted. This function is passed to the
+ *  [@apify/http-request](https://www.npmjs.com/package/@apify/http-request) NPM package.
  */
 
 /**

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -4,6 +4,10 @@
         "module": "commonjs",
         "moduleResolution": "node",
         "lib": [
+            "DOM",
+            "DOM.Iterable",
+            "ES2015",
+            "ES2016",
             "ES2018",
             "ES2019.Object",
             "ES2018.AsyncIterable"

